resources/states.raml

```diff
@@ -87,3 +87,54 @@ post:
         body:
           application/json:
             example: !include ../examples/state.example.json
+/key={key}:
+  (methodName): withKey
+  type:
+    baseResource:
+      uriParameterName: Key
+      resourceType: State
+      resourceUpdateType: Update
+  get:
+    securedBy:
+      [
+        oauth_2_0:
+          {
+            scopes: ['manage_project:{projectKey}', 'view_orders:{projectKey}'],
+          },
+      ]
+    responses:
+      200:
+        body:
+          application/json:
+            example: !include ../examples/state.example.json
+  post:
+    securedBy:
+      [
+        oauth_2_0:
+          {
+            scopes:
+              ['manage_project:{projectKey}', 'manage_orders:{projectKey}'],
```

Comment:
the same here you should add "manage_states:{projectKey}", because "manage_orders:{projectKey}" is the deprecated one. See https://docs.commercetools.com/http-api-projects-states#update-state-by-key